enterprise/reporting/error_index/error_index_reporting.go

```diff
@@ -305,8 +305,11 @@ func (eir *ErrorIndexReporter) mainLoop(ctx context.Context, errIndexDB *jobsdb.
 }
 
 func (eir *ErrorIndexReporter) Stop() {
+	eir.log.Infow("Stopping error index reporter")
 	eir.cancel()
-	_ = eir.g.Wait()
+	if err := eir.g.Wait(); err != nil {
+		eir.log.Errorw("Error index reporter failed to stop", "err", err)
```

Comment:
wouldn't say `failed to stop` imo.
It did stop, albeit with some underlying issue